Changelog.md

```diff
@@ -14,6 +14,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `with_connections` has been replaced by `with_servers` which contains the connection metadata (increment_by, timeout, allocations, etc) (https://github.com/zendesk/global_uid/pull/71)
 - The `per_process_affinity` configuration has been replaced with `connection_shuffling`. The behaviour remains the same, but the default boolean value has flipped – e.g. if you had previously configured `per_process_affinity = false`, you should now set `connection_shuffling = true` to get the same behavior. `connection_shuffling` defaults to false.
 - [Breaking change] Move configuration to a class
+- [Breaking change] `notifier` configuration option now only takes an exception, rather than exception and message
```

Comment:
```suggestion
- [Breaking change] The configured `notifier` proc is now called now only with an exception, rather than an exception and a message.
```